.github/workflows/start-release-train.yml

```diff
@@ -13,10 +13,65 @@ on:
 env:
   LC_ALL: en_US.UTF-8
   LANG: en_US.UTF-8
+concurrency:
+  group: start-release-train-${{ inputs.version_number }}
+  cancel-in-progress: true
 jobs:
-  placeholder:
-    name: Placeholder job
-    runs-on: ubuntu-latest
+  create_release_pr:
+    name: Create release branch and bump version
+    runs-on: macos-latest
+    outputs:
+      branch_name: ${{ steps.branching.outputs.branch_name }}
     steps:
-      - name: Placeholder step
-        run: echo "Placeholder step"
+      - uses: actions/checkout@v4
+
+      - name: Setup Git User
+        run: |
+          # setup git
+          git config --global user.name "Bitmovin Release Automation"
+          git config --global user.email "support@bitmovin.com"
+
+      - name: Setup node and npm registry
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          registry-url: 'https://registry.npmjs.org/'
+          cache: 'yarn'
+
+      - name: Set Release Branch name
+        id: branching
+        run: |
+          branch_name="release/v${{ inputs.version_number }}"
+          echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
+
+      - name: Create Release Branch
+        run: |
+          git push origin --delete ${{ steps.branching.outputs.branch_name }} || true
```

Comment:
Why do we need to use --delete here? Aren't we creating a new branch here?

Reply:
Link to the answer: https://github.com/bitmovin/bitmovin-player-react-native/pull/223#discussion_r1318514325

I see, might be worth a comment 🙂

Reply:
Adding it here, I don't know how it ended up being a separate comment 😓 

In case we need to re-run this job it will delete the existing remote branch and start over.